cinnabar/wrangle.py

```diff
@@ -58,85 +65,35 @@ def toDF(self):
         )
 
 
-class ExperimentalResultFromCSV(object):
-    def __init__(self, ligand, expt_DG, expt_dDG):
+class ExperimentalResult:
+    def __init__(self, ligand: str, expt_DG: float, expt_dDG: float):
         self.ligand = ligand
-        self.DG = float(expt_DG)
-        try:
-            error = expt_dDG.strip("\n")
-        except AttributeError:
-            # Data is already numeric
-            error = expt_dDG
-        self.dDG = float(error)
-
-
-class FEResults(object):
-    """
-    Wrapper class to store results from Free Energy calculations.
-
-    Useful for programatically creating a container to pass to FEMap class.
-    """
-
-    def __init__(self):
-        """Create results object"""
-        # Initializing attributes
-        self.experimental_results = dict()
-        self.calculated_results = list()
-        self.fe_results = {
-            "Experimental": self.experimental_results,
-            "Calculated": self.calculated_results,
-        }
-
-    def add_experimental_obj(self, experimental_result):
-        """Adds experimental result from ExperimentalResult object."""
-        self.experimental_results[experimental_result.ligand] = experimental_result
-
-    def add_calculated_obj(self, calculated_result):
-        """Adds calculated result from CalculatedResult object."""
-        self.calculated_results.append(calculated_result)
-
-    def add_experimental_result(self, ligand, expt_value, expt_error):
-        """
-        Add experimental result from plain explicit values.
+        self.DG = expt_DG
+        self.dDG = expt_dDG
 
-        Examples
-        --------
-        >>> fe_results = FEResults()
-        >>> fe_results.add_experimental_result("CAT-13a", -8.93, 0.10)
-        """
-        self.experimental_results[ligand] = ExperimentalResultFromCSV(
-            ligand, expt_value, expt_error
-        )
 
-    def add_calculated_result(self, ligand_a, ligand_b, calc_value, mbar_error, other_error):
-        """
-        Add calculated relative result from plain explicit values.
+class FEMap:
+    results: list[Union[RelativeResult, ExperimentalResult]]
+    computational_graph: nx.MultiDiGraph
+    experimental_graph: nx.MultiDiGraph
```

Comment:
This seemed like the nicest approach... So two graphs, one is bench measurements, one computational measurements.  Both Multi graphs as you could have multiple measurements of the same node/edge (todo: not sure if NX allows multiple identical nodes (i.e. multiple absolute measurements))

Reply:
so a way to do multiple absolute measurements is to have **all** measurements as edges, and absolute measurements are from a dummy node (I used `"NULL"`)

Reply:
Would it make sense (maybe longer term) to just have arbitrary numbers of graphs here?

I.e. if I have results from ITC, NMR, etc... and then some calculated metrics, I'd like to just dump them all in the same class and do all my analyses at the same time?

Reply:
could do...  I think I'm going to allow arbitrary annotations per measurement (which end up as edge annotations on the graph), this is where e.g. `mbar_err` can live.  Then on the experimental graph you could have `source="ITC"` on some measurements.

Though that's also an argument you could have one graph, with edges first annotated as `type='computational'`

Reply:
> Though that's also an argument you could have one graph, with edges first annotated as type='computational'

Pretty much :P 

Also technically we don't need to treat computational vs experimental differently, we just need to know what we're using as our ground truth and what we're adjusting, so you could just go "label A is experiment, label B is calculated", etc...